src/olympia/amo/tests/test_utils_.py

```diff
@@ -172,3 +175,19 @@ def bar(self):
     del foo.bar
     assert foo.bar == 'original value'
     assert callme.call_count == 1
+
+
+@pytest.mark.parametrize('lang', settings.AMO_LANGUAGES)
+def test_get_locale_from_lang(lang):
+    """Make sure all languages in settings.AMO_LANGUAGES can be resolved."""
+    locale = get_locale_from_lang(lang)
+
+    assert isinstance(locale, Locale)
+    assert locale.language == lang[:2]
```

Comment:
Somewhat. My concern is that the `Locale` object we return is completely different from before for a lot of languages (as shown by the difference in the value returned by `locale.language`), so I wonder if it has consequences later down the road.